shenyu-plugin/shenyu-plugin-httpclient/src/main/java/org/apache/shenyu/plugin/httpclient/WebClientPlugin.java

```diff
@@ -49,27 +50,29 @@ public WebClientPlugin(final WebClient webClient) {
     }
 
     @Override
-    protected Mono<ClientResponse> doRequest(final ServerWebExchange exchange, final String httpMethod, final URI uri,
+    protected Mono<ResponseEntity<Void>> doRequest(final ServerWebExchange exchange, final String httpMethod, final URI uri,
```

Comment:
if modify this， may be have modify this : WebClientMessageWriter

Reply:
Agree @yu199195 , I was missing this piece. we can modify the WebClientMessageWriter to use newer WebClient.ResponseSpec instead of ClientResponse to process data. what are your thoughts on the above.